api/util/errorLog-function.js

```diff
@@ -9,13 +9,13 @@ const ErrorLog = require('../models/ErrorLog');
  * @param {string} req.body.apiEndpoint - The location of the error
  * @param {string} req.body.errordescription - The description of the error
  */
-async function addErrorLog(req){
+async function addErrorLog(userEmail, errorTime, apiEndpoint, errorDescription){
```

Comment:
rename the **file** to `ErrorLog.js`

Reply:
going to rename it errorLog.js because that is the format for the other files in the folder